assets/charts/components/contour/templates/02-job-certgen.yaml

```diff
@@ -47,10 +47,6 @@ spec:
     spec:
       containers:
       - name: contour
-        # This version is set to latest because Job specs are immutable;
```

Comment:
Why do we remove this comment? It seems still relevant?

Reply:
Rest of the comment should also be restored IMO. @surajssd ?

Reply:
If upstream has removed it for whatever reasons then why keep it?

Reply:
Ah, I thought it was our modification. If we follow upstream here, then sure, let's remove it :+1: